docs/DEPLOYMENT.MD

```diff
@@ -0,0 +1,28 @@
+# DEPLOYMENT
+This document covers steps on setting up this repository on various cloud hosting providers.
+
+# AWS
+
+# Staging
+1. Login to your aws console.
+2. Create a S3 bucket and uncheck `Block all plublic access`.
+3. Enable static website hosting and note down the bucket endpoint. We'll use it to set up a Cloudfront distribution later.
+4. In Cloudfront, create a distribution with the S3 origin, created in the step above.
+5. If you choose to set up a SSL, do so at this step.
+6. Use a DNS provider of your choice. Give an appropriate CNAME and enter the target address. This would be the address of the newly created Cloudfront distribution.
+7. Next, go to your GitHub repository and create a new environment from settings tab.
+        i. Name the environment as Staging.
+        ii. Make sure you have added the following GitHub secrets on repository level. If not, add these as your environment secrets.
+                AWS_ACCESS_KEY_ID
+                AWS_SECRET_ACCESS_KEY
+                AWS_REGION
+        iii. Additionally, refer to .env.example file to set all the other required environment variables for your project.
+
+8. Add the following lines to the `scripts` object in `package.json`.
+    - "deploy-staging": "vue-cli-service build --mode staging && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGION s3 sync ./dist $S3_STAGING_BUCKET_NAME --delete --acl public-read",
+    "deploy": "vue-cli-service build && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGION s3 sync ./dist $S3_PROD_BUCKET_NAME --delete --acl public-read"
```

Comment:
<img width="995" alt="Screenshot 2021-08-05 at 4 01 00 PM" src="https://user-images.githubusercontent.com/20169829/128335788-74b60d46-d1a4-4922-bb1c-dde0d94ae539.png">
This looks like as shown in the screenshot in markdown
Better way to write this would be using code blocks in markdown
See this https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks